repo2docker/buildpacks/r.py

```diff
@@ -178,6 +180,7 @@ def get_packages(self):
         # install from a different PPA
         if V(self.r_version) < V("3.5"):
             packages.append("r-base")
+            packages.append("r-base-dev")
```

Comment:
```suggestion
            packages.append("r-base-dev")
            packages.append("libclang-dev")
```

seems like we also need `libclang-dev` now